oct/ansible/oct/playbooks/package/ami-mark-ready.yml

```diff
@@ -59,3 +59,12 @@
         tags: "{ '{{ item.key }}': '{{ item.value }}' }"
         state: present
       with_dict: "{{ origin_ci_aws_additional_tags }}"
+
+    - name: grant launch permissions
+      ec2_ami:
+        image_id: '{{ origin_ci_aws_ami_id }}'
+        region: '{{ origin_ci_aws_region }}'
+        state: present
+        launch_permissions:
+          user_ids: '{{ aws_account_ids_for_launch_permissions }}'
+      when: '{{ aws_account_ids_for_launch_permissions }}'
```

Comment:
Can you change this to a more explicit check? Like the length is not 0 or something similar, not sure about what the default truthy `when` does

Reply:
Yeah, I didn't realize Ansible had such stupid ideas about what's True and False :(

Working on this now

Reply:
Also Python in general :)